include/openPMD/IO/ADIOS/ParallelADIOS1IOHandlerImpl.hpp

```diff
@@ -94,6 +94,7 @@ namespace openPMD
         std::unordered_map< std::shared_ptr< std::string >, ADIOS_FILE* > m_openReadFileHandles;
         std::unordered_map< ADIOS_FILE*, std::vector< ADIOS_SELECTION* > > m_scheduledReads;
         std::unordered_map< int64_t, std::unordered_map< std::string, Attribute > > m_attributeWrites;
+        int64_t GetFileHandle(Writable*);
```

Comment:
Can you please add a doxygen string that describes the side-effects and return-value of this function?
We call this in several places without handling the return value, which might lead to confusion later on.

Reply:
added